example.json

```diff
@@ -15,6 +15,7 @@
     "mobileInstruction": "This is optional instruction text that will be shown when viewed on mobile.",
     "_setCompletionOn":"allItems",
     "_useGraphicsAsPins": false,
+    "_useNumbersAsPins": false,
```

Comment:
Would custom pins fall into this scope? https://github.com/adaptlearning/adapt_framework/issues/1574#issuecomment-306441819